### ISSUE TITLE
Higher order modes implementation

### ISSUE THREAD

**Issue by ppandey10** [ppandey10/mlgw_bns#1]
Training my model for each mode might be a little time consuming, so will try to include all the significant modes and then work with that. 

Right now, I am not very confident with the former approach, but will definitely try that too.

**Comment by jacopok** [comment]
I think the issue of the time consumption in training a model for all modes might be relevant in the future, but for the moment the more pressing concern is: can we train a model to reproduce _any_ modes and recover a sensible waveform?

Step 1 could be to do it with simply the (2, 2) mode (should recover the previous results), then the simplest extension could be to do (2, 2) + another mode like the (3, 3).

One thing at a time :D

P.S. what do you mean by "all the significant modes"?

**Comment by ppandey10** [comment]
Yes, will train my model for (2, 2) in a day or two. By significant modes, I meant the "dominance" of individual modes, if that makes sense? Therefore, the next obvious choice is to work with (2, 2) + (3, 3)

**Comment by jacopok** [comment]
Yes, I think we're on the same page!
